services/user/CommonSys/common/packages/wasm-loader/src/lib/plugin.js

```diff
@@ -0,0 +1,123 @@
+import { sleep } from 'yootils';
+
+export let packagesUrl = 'https://unpkg.com';
+
+const fetch_cache = new Map();
+
+async function fetch_if_uncached(url) {
+	if (fetch_cache.has(url)) {
+		return fetch_cache.get(url);
+	}
+
+	await sleep(200);

```

Comment:
This is a local copy of the [https://github.com/DougAnderson444/rollup-plugin-wit-component](rollup-plugin-wit-component) repo. I added a local copy so we could control the version of JCO instead of having to wait for Doug to publish a new version of his plugin. Furthermore we will eventually cache the transpilation in this plugin for some additional time savings.

I've now added attribution to the tops of all .js files in this `/lib/` directory.